inst/site/01-results-by-iteration.Rmd

```diff
@@ -1,4 +1,4 @@
-# Iteration specific results
+# Month specific results
```

Comment:
I did not bother much about this, but then you may want to also rename the name of the file.